frontend/src/routes/AdminPageAbakusLeaderView/index.jsx

```diff
@@ -115,21 +113,21 @@ const AdminPageAbakusLeaderView = () => {
           <Statistics>
             <StatisticsWrapper>
               <StatisticsName>Søknader åpner</StatisticsName>
-              <Moment format="HH:mm:ss dddd Do MMMM">
+              <FormatTime format="HH:mm:ss dddd Do MMMM">
                 {admission.open_from}
-              </Moment>
+              </FormatTime>
             </StatisticsWrapper>
             <StatisticsWrapper>
               <StatisticsName>Søknadsfrist</StatisticsName>
-              <Moment format="HH:mm:ss dddd Do MMMM">
+              <FormatTime format="HH:mm:ss dddd Do MMMM">
                 {admission.public_deadline}
-              </Moment>
+              </FormatTime>
             </StatisticsWrapper>
             <StatisticsWrapper>
               <StatisticsName>Redigeringsfrist</StatisticsName>
-              <Moment format="HH:mm:ss dddd Do MMMM">
+              <FormatTime format="HH:mm:ss dddd Do MMMM">
                 {admission.application_deadline}
-              </Moment>
+              </FormatTime>
```

Comment:
A couple of these seem to have changed as well;
![image](https://user-images.githubusercontent.com/13599770/187073600-e0ea1451-0f82-441b-863e-d38afaa06838.png)